client/src/commponet/Description.js

```diff
@@ -0,0 +1,93 @@
+import { useState } from "react";
+// import "bootstrap/dist/css/bootstrap.min.css";
+import { Container, Form, Row, Col } from "react-bootstrap";
+import  Button from "react-bootstrap/Button";
+
+
+const Description = () =>{
+const [addingEnergiser, setAddingEnergiser] = useState(false);
+const [energiserData, setEneriserData] = useState(null);
+// const [energiserId, setEnergiserId] = useState();
+const [name, setName] = useState("");
+const [description, setDescription] = useState("");
+
+
+const handleEnergiserAdder = (e) => {
+    e.preventDefault();
+
+
+        let energiser = {
+      name: name,
+      description: description,
+    };
+
+     fetch("/api/energizers",{
+            method:"POST",
+            headers:{ "Content-Type": "application/json" },
+            body: JSON.stringify(energiser),
+          })
+          .then((res) => res.json())
+          .then((data) => {
+            console.log(data);
+            //  setEneriserData(data);
+          }).
+          catch((error)=> {
+        console.log(error.message);
+    });
+    setEneriserData([...energiserData, energiser ]);
+    setName("");
+    setDescription("");
+};
```

Comment:
I think basic validation should be added that prevents submission of empty fields